pilot/pkg/networking/core/v1alpha3/accesslog.go

```diff
@@ -339,6 +339,7 @@ func buildTCPGrpcAccessLog(isListener bool) *accesslog.AccessLog {
 					},
 				},
 			},
+			TransportApiVersion:     core.ApiVersion_V3,
```

Comment:
Do we need to version guard this? or it will work for 1.8 proxies also?

Reply:
I just tested, this works with 1.8 proxy.

Reply:
Is this fix related to Istio 1.9 error https://github.com/istio/istio/issues/30939?
Because I am also getting the same error

Reply:
@ms8527 The fix will be available in `1.9.1`.

Reply:
@shamsher31 Thanks for the response. Any idea when the 1.9.1 release will be available? Also, is there any other way in 1.9 version to tell the gRPC service where to emit the logs? Since, enabling **meshConfig.enableEnvoyAccessLogService** isn't working currently with Istio 1.9.